Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.10.5-slim-bullseye
```

Python's Alpine image (17.66 MB) is more than half the size of the slim-bullseye image (45.2 MB), so I would rather use that.

I do not know Python's docker images that well, so if there is a bigger benefit than size reduction, I am happy to adopt the slim-bullseye image.

On a second note, I would leave out the patch version number as well, as there are no breaking changes/feature updates in these releases, and it should therefore always use the latest.

```suggestion
FROM python:3.10-alpine
```